gm4_liquid_tanks/data/gm4_standard_liquids/function/util/honey_casting/honey_cauldron.mcfunction

```diff
@@ -2,7 +2,20 @@
 #run from standard_liquids:util_above
 
 execute if score @s gm4_lt_util matches 4.. run scoreboard players remove @s gm4_lt_value 4
-execute if score @s gm4_lt_util matches 4.. run summon armor_stand ~0.5 ~ ~0.5 {Marker:1b,Invisible:1b,Invulnerable:1b,NoGravity:1b,Silent:1b,Fire:4000s,Tags:["gm4_lt_honey_display","gm4_no_edit"],CustomName:'"gm4_lt_honey_display"',Pose:{Head:[180.0f,0.0f,0.0f]},ArmorItems:[{},{},{},{id:"minecraft:honey_block",count:1,components:{"minecraft:custom_model_data":"block/cast_honey_block"}}],DisabledSlots:4144959}
+execute if score @s gm4_lt_util matches 4.. run summon item_display ~0.5 ~ ~0.5 :
+    Tags:["gm4_lt_honey_display","gm4_no_edit"]
+    CustomName:'"gm4_lt_honey_display"'
+    item:
```

Comment:
I'm not a fan of using this syntax, because you lose language server support and it becomes harder to auto-update certain fields in the future. With line wrapping enabled it's really not a huge deal to have long NBT lines.

Reply:
Even with line wrapping, I find long NBT lines super obnoxious to read/deal with, especially when things start getting nested like they are so often with item displays. Bolt already has an incompatibility with the spyglass language server (I still need to try out the new mecha one) so this syntax isn't the sole offender. I also think except in cases where you're auto-updating complex compounds, this should still play nice with regex find/replace for the most common changes like capitalizing keys or something.

Reply:
Ehh, I still disagree, but I won't fight you on this. Imo 

> Bolt already has an incompatibility with the spyglass language server

This is just a mcfunction file though, that otherwise would've been completely valid in vanilla.